app/uk/gov/nationalarchives/omega/editorial/support/DisplayedCreator.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2022 The National Archives
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package uk.gov.nationalarchives.omega.editorial.support
+
+import uk.gov.nationalarchives.omega.editorial.models.Creator
+import uk.gov.nationalarchives.omega.editorial.models.Creator.{ CreatorTypeCorporateBody, CreatorTypePerson }
+
+case class DisplayedCreator(id: String, name: String)
```

Comment:
Do need this case class, or could `fromCreator` be a method on  `Creator`? Possibly called `formattedName` or some such?

Reply:
I'm afraid I don't understand; could you sketch it out?

Reply:
Now, I don't think we'd ever need to convert a `DisplayedCreator` into a `Creator`; it's just for presentation.

Reply:
I understand it now; let me look into it, @felixgb.

Reply:
Yes, good idea - `DisplayedCreator` is unnecessary.

Reply:
Changed.